Assets/PlayerController.cs

```diff
@@ -35,4 +36,14 @@ void OnCollisionEnter2D(Collision2D collision)
             canJump = true;
         }
     }
+
+    public void TakeDamage()
```

Comment:
Should probably intake a float of the damage that we are taking. Since an external piece of code is calling this, it will probably know what amount of damage the player should take (same goes for the AI). This would also obligate a "Damage" property on this class to pass when an attack is called. I know this is just the first iteration of this, so we can consider adding this as an issue for later on, but it feels really easy to add on to this.